pkgs/applications/virtualization/arion/update

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash
+#!nix-shell -p cabal2nix
+
+#
+# Usage ./update ~/src/arion
+#
+# Make sure that ~/src/arion matches the latest version on hackage.
```

Comment:
I don't think this necessarily needs to be changed, but it might be easier for other people to run if this update script just pulled `arion` from the latest version on Hackage and used that.  Then you wouldn't need `arion` checked out locally (and have to match versions by hand).